tests/functional/restart/50-two-flows/flow.cylc

```diff
@@ -21,8 +21,9 @@
     [[c]]
        script = """
            if ((CYLC_TASK_FLOW_NUMBERS == 1)); then
-               cylc trigger --flow=new --meta="cheese wizard" "$CYLC_WORKFLOW_NAME//1/a"
+               cylc trigger --flow=new --meta="cheese wizard"
+               "$CYLC_WORKFLOW_ID//1/a"
```

Comment:
```suggestion
               cylc trigger --flow=new --meta="cheese wizard" \
                   "$CYLC_WORKFLOW_ID//1/a"
```

Reply:
Ah, thanks @wxtim ; looks like I had an editor auto line break fail there after changing NAME to ID in the test.